python/hsfs/feature_group.py

```diff
@@ -1363,7 +1363,7 @@ def _enable_feature_monitoring(
             job_frequency=job_frequency,
             start_date_time=start_date_time,
             is_event_time=False,
-            training_dataset_version=None,
+            transformed_with_version=None,
```

Comment:
Do we really need `transformed_with_version` for feature groups?

Reply:
Just making sure it is forced to None, it is not part of the public api for FG